distributions/validate.py

```diff
@@ -0,0 +1,60 @@
+import requests
```

Comment:
This script is useful to quickly validate the urls. It doesn't have to be checked in, but it would be nice to have